src/test/ui/resolve/use_suggestion_placement.stderr

```diff
@@ -18,19 +18,7 @@ help: possible candidate is found in another module, you can import it into scop
 LL | use m::A;
    |
 
-error[E0412]: cannot find type `HashMap` in this scope
-  --> $DIR/use_suggestion_placement.rs:27:23
-   |
-LL |     type Dict<K, V> = HashMap<K, V>;
-   |                       ^^^^^^^ not found in this scope
-help: possible candidates are found in other modules, you can import them into scope
-   |
-LL | use std::collections::HashMap;
-   |
-LL | use std::collections::hash_map::HashMap;
-   |
-
-error: aborting due to 3 previous errors
+error: aborting due to 2 previous errors
```

Comment:
This test is supposed to test the `use x::y::z;` suggestion, which is now gone. Replacing `HashMap` with `BTreeMap` should fix this.